tools/roslaunch/scripts/roscore

```diff
@@ -59,6 +59,16 @@ def _get_optparse():
     parser.add_option("--master-logger-level",
                       dest="master_logger_level", default=False, type=str,
                       help="set rosmaster.master logger level ('debug', 'info', 'warn', 'error', 'fatal')")
+    parser.add_option("--sigint-timeout",
+                      dest="sigint_timeout",
+                      default=15, type=float,
```

Comment:
Why are the constants repeated here rather then included?

Same below for sigterm.